File: Php/app/Cashgame/Listing/CashgameListingModel.php
<?php
namespace app\Cashgame\Listing{

	use app\Cashgame\CashgameNavigationModel;
	use core\HomegamePageModel;
	use entities\Cashgame;
	use entities\Homegame;
	use Domain\Classes\User;
	use app\Cashgame\Listing\CashgameTable\CashgameTableModel;

	class CashgameListingModel extends HomegamePageModel {

		public $listTableModel;
		public $cashgameNavModel;

		public function __construct(User $user,
									Homegame $homegame,
									array $cashgames,
									array $years = null,
									$year = null,
									Cashgame $runningGame = null){
			parent::__construct($user, $homegame, $runningGame);
			$this->listTableModel = new CashgameTableModel($homegame, $cashgames);
			$this->cashgameNavModel = new CashgameNavigationModel($homegame, 'listing', $years, $year, $runningGame);
		}

	}

}

File: Web/Models/CashgameModels/Listing/CashgameListingModel.cs
using System.Collections.Generic;
using Core.Classes;
using Web.Models.NavigationModels;
using Web.Models.PageBaseModels;

namespace Web.Models.CashgameModels.Listing{

	public class CashgameListingModel : HomegamePageModel {

		public CashgameTableModel ListTableModel;
		public CashgameNavigationModel CashgameNavModel;

		public CashgameListingModel(User user,
									Homegame homegame,
									List<Cashgame> cashgames,
									List<int>  years,
									int? year,
									Cashgame runningGame)
            : base(user, homegame, runningGame)
        {
			ListTableModel = new CashgameTableModel(homegame, cashgames);
			CashgameNavModel = new CashgameNavigationModel(homegame, "listing", years, year, runningGame);
		}

        public override string BrowserTitle
        {
            get { return "Cashgame List"; }
        }

	}

}